.github/CODEOWNERS

```diff
@@ -16,6 +16,7 @@ CMakeLists.txt @tlrmchlsmth
 
 # vLLM V1
 /vllm/v1 @WoosukKwon @robertgshaw2-redhat @njhill @ywang96 @comaniac @alexm-redhat
+/vllm/v1/struct_output @aarnphm @russellb
```

Comment:
I think this might not work for people that are not maintainers.

Even if it does, I think it's a bit confusing to list non-maintainers in here.

Reply:
I was renaming file paths and this line came up in my search, so I removed it for now -- can re-add it if the maintainers think it makes sense though